amqp-protocol-binding.md

```diff
@@ -94,33 +89,14 @@ mandate specific existing features to be used.
 This specification does not further define any of the [CloudEvents][ce] event
 attributes.
 
-Two of the event attributes, `datacontenttype` and `data` are handled specially
-and mapped onto AMQP constructs, all other attributes are transferred as
+One event attribute, `datacontenttype` is handled specially in *binary* content
+mode and mapped onto AMQP constructs. All other attributes are transferred as
```

Comment:
"mapped on to the AMQP content-type message property" - "AMQP constructs" is needlessly vague, it's clearer to be explicit.